packages/mobile/src/hooks/useReachabilityEffect.ts

```diff
@@ -22,7 +26,11 @@ export const useReachabilityState = (
         console.info('Become reachable')
         onBecomeReachable()
       }
-      if (!nextReachabilityState && prevReachability && onBecomeUnreachable) {
+      if (
+        !nextReachabilityState &&
+        (prevReachability || prevReachability === undefined) &&
```

Comment:
@nicoback I think this was the root cause of lineups not loading in offline mode. We weren't firing the `onBecomeUnreachable` callback initially because prevReachability was undefined